src/middlewares/approveOrReject.js

```diff
@@ -0,0 +1,42 @@
+import { users as user, userProfile as userProfiles, tripRequests } from '../database/models';
+import Customize from '../helpers/Customize';
+
+const isManagerAccess = async (req, res, next) => {
+  const { id } = req.user;
+  const getRole = await user.findAll({ where: { roleId: 6, id } });
+  const [{ dataValues }] = getRole;
+  return dataValues.roleId !== 6
+    ? Customize.successMessage(req, res, 'You do not have access to perform this action as a manager', 403) : next();
+};
+
+const validateAcceptOrReject = async (req, res, next) => {
+  const { id } = req.user;
+  const { tripRequestId } = req.params;
+  const users = await userProfiles.findAll({ where: { managerId: id } });
+  const newArray = [];
+  const allUserId = users.map((i) => newArray.push(i));
+
+  const findRequest = await tripRequests.findOne({
+    where: { userId: allUserId, id: tripRequestId }, raw: true
+  });
+
+  const { statusId } = findRequest;
+  const { status } = req.query;
+
+  if (status === 'reject') {
+    if (statusId === 3) {
+      return Customize.errorMessage(req, res, 'this request already rejected', 409);
```

Comment:
Just grammatical accuracy, I think it should be `This request is already rejected`